### DISCUSSION TITLE
Einbinden der Pflichtfelder BT-23, BT-34 und BT-49

### DISCUSSION THREAD

**Discussion by Jericho699** [stephanstapel/ZUGFeRD-csharp#422] [Q&A]
Hallo,
wie kann ich die Pflichtfelder BT-23, BT-34 und BT-49 einbinden?
Es wäre super wenn ich dazu eine Unterstützung bekommen könnte.

**Comment by stephanstapel** [comment]
In vielen Fällen ist es hilfreich, wenn Du Quellcode nach den Bezeichnern durchsuchst, das geht auf github ja ganz bequem.
